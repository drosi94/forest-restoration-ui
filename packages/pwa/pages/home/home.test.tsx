import React from 'react'
import { render } from '../../test/testUtils'
import Home from './index.page'

describe('Home page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Home />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
