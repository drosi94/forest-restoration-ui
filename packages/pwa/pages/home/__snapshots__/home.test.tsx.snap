// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-1pcxayg-Home"
  >
    <div
      class="css-164z9z8-Home"
    >
      <button
        class="css-z3eb22-Button e1p27jrl0"
      >
        common:hello
      </button>
      <h1
        class="css-nhbcyc-Typography-Home e1eqcpp10"
      >
        home:test
      </h1>
      <button
        class="css-z3eb22-Button e1p27jrl0"
      >
        open
      </button>
      <div
        class="css-163aj9c-Home"
      >
        <button
          aria-expanded="false"
          class="css-1ulw4zu-Accordion"
          id="headlessui-disclosure-button-1"
          type="button"
        >
          <h2
            class="css-gxuocn-Typography-Accordion e1eqcpp10"
          >
            Hello there
          </h2>
          <svg
            class="css-tcqes-Accordion"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M14.707 12.707a1 1 0 01-1.414 0L10 9.414l-3.293 3.293a1 1 0 01-1.414-1.414l4-4a1 1 0 011.414 0l4 4a1 1 0 010 1.414z"
              fill-rule="evenodd"
            />
          </svg>
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
